src/components/ProjectCard.js

```diff
@@ -0,0 +1,29 @@
+import React from "react";
+// import Project from "../components/Project";
+import "./ProjectCard.css";
+
+import { Route, Link } from "react-router-dom";
+
+export default function ProjectCard(props) {
+    const { year, title, img, technologie, bgColor } = props;
+    return (
+        <div
+            style={{ backgroundColor: bgColor }}
+            className="project-card-container"
+        >
+            <div className="project-card-info">
+                <p>{year}</p>
+                <strong>
+                    <h2>{title}</h2>
+                </strong>
+                <p>Full Stack Development</p>
+            </div>
+            <picture className="project-card-picture">
+                <Link to={`/project/${title}`}>
+                    <img className="project-img" src={img} alt="project" />
```

Comment:
your `alt` should be meaningful for a screen reader .. you can use the title of the project instead